packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/About.5.doc.mdx

```diff
@@ -30,51 +103,47 @@ This component requires the following peer dependencies be installed in your app
 import { FlowsheetDataGrid } from 'terra-data-grid';
 ```
 
-## Component Features
-
- * [Cross-Browser Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#cross-browser-support)
- * [Mobile Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#mobile-support)
- * [Internationalization Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#internationalization-i18n)
- * [LTR/RTL Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#ltr--rtl)
-
-## Examples
-|Link to Example| Description|
-|-|-|
-|[BasicFlowsheetDataGrid](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/examples/basic-flowsheet-data-grid)|An example of the Flowsheet Data Grid with basic keyboard interactions.|
-|[NoResultsCells](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/examples/no-result-cells)|An example containing "No result" cells.|
-|[CellSelection](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/examples/cell-selection)|An example demonstrating the ability to select a single cell or range of cells.|
-|[ColumnHeadersHidden](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/examples/column-headers-hidden)|An example containing visually hidden column headers.|
-|[RowSelection](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/examples/row-selection)|An example demonstrating the ability to select a row of cells.|
+## Properties
+<Notice variant="important" ariaLevel="2">
+When making changes to the columns, section, rows, or cells, the Flowsheet Data Grid component requires you to pass new objects so that React can detect the changes.  
+In addition, ensure that you memoize callback functions using methods such as the useCallback hook to prevent unnecessary rerenders of components.
+</Notice>
 
-## Grid props table
+### Flowsheet Data Grid Property Table
 <FlowsheetDataGridPropsTable />
 
 ### Column
 A column specifies the data to render a cell in the header row of the Flowsheet Data Grid.
 
 |Name|Type|Is Required|Default Value|Description|
 |---|---|---|---|---|
-|**id**|string|<span style={{color:'#d53040'}}>required</span>|none|An identifier to uniquely identify the column within the grid.|
-|**displayName**|string|optional|none|String of text to render within the column header cell.|
-|**hasError**|bool|optional|none|Boolean value indicating whether or not the column has an error in the data.|
-|**width**|number|optional|none|A number (in px) specifying the width of the column. If not provided, the Data Grid's default column width will be used.|
+|**id**|string|<span style={{color:'#d53040'}}>required</span>|none|An identifier to uniquely identify the column in the grid.|
```

Comment:
nit: ```suggestion
|**id**|string|<span style={{color:'#d53040'}}>required</span>|none|An identifier to uniquely identify a column in the grid.|
```

Reply:
This was updated by the technical document developer so I hesitate to change the language.

Reply:
@nikhitasharma  @cm9361  I think this is probably okay as is.  The prop is within a section on the page specific to Columns, with the assumption each column on the grid will have it's on respective props.  So I think the usage of the word "the" in this instance works.